src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -175,6 +180,28 @@ export default function ConfirmedAppointmentDetailsPage() {
         This appointment occurred in the past.
       </InfoAlert>
     );
+  } else if (showConfirmMsg) {
+    infoAlert = (
```

Comment:
This is the alert from the confirmation page, copied to here now that we're getting an appointment id back from the submit and can redirect to the detail page.